### PR TITLE
feat: json -> Args -> abi

### PR DESCRIPTION
**Pull request by JoowonYun**
Support convert json -> args -> abi.

I have used abi directly before.
Now, users can more readily available using the json.

### CODE REVIEW

**Comment by jhyeom26** [comment]
https://github.com/golang/protobuf/issues/193

According to the issue, we can change CamelCase to snake_case~!

File: util/util.go [hunk 1]
```diff
@@ -138,6 +142,96 @@ func AbiBigIntTobytes(src *big.Int) []byte {
...
+		data = AbiUint64ToBytes(uint64(src.GetLongValue()))
+	case *consensus.Deploy_Arg_Value_BigInt:
+		// TODO : ParseUint didn't support 512.. need to change more.
+		bitWidth := uint32(64)
+		if src.GetBigInt().GetBitWidth() < 64 {
+			bitWidth = src.GetBigInt().GetBitWidth()
+		}
+		val, err := strconv.ParseUint(src.GetBigInt().GetValue(), 10, int(bitWidth))
+		if err != nil {
+			return []byte{}
```
Comment (jhyeom26) [inline util/util.go]: Why don't we propagate the error?

File: util/util.go [hunk 2]
```diff
@@ -138,6 +142,96 @@ func AbiBigIntTobytes(src *big.Int) []byte {
...
+			data = append([]byte{WASM}, src.GetKey().GetAddress().GetAccount()...)
+		case *state.Key_Hash_:
+			data = append([]byte{HASH}, src.GetKey().GetHash().GetHash()...)
+		case *state.Key_Uref:
+			data = append([]byte{UREF}, src.GetKey().GetUref().GetUref()...)
+			data = append(data, []byte{byte(src.GetKey().GetUref().GetAccessRights())}...)
+		case *state.Key_Local_:
+			data = append([]byte{LOCAL}, src.GetKey().GetLocal().GetHash()...)
+		}
+	default:
```
Comment (jhyeom26) [inline util/util.go]: same here

File: util/util.go [hunk 3]
```diff
@@ -138,6 +142,96 @@ func AbiBigIntTobytes(src *big.Int) []byte {
...
+	res = append(res, data...)
+
+	return res
+}
+
+func JsonStringToDeployArgs(str string) (deployArgs []*consensus.Deploy_Arg, err error) {
+	jsonDecoder := json.NewDecoder(strings.NewReader(str))
+	_, err = jsonDecoder.Token()
+	if err != nil {
+		return deployArgs, err
```
Comment (jhyeom26) [inline util/util.go]: Wouldn't it be more clearer returning `nil, err` than `deployArgs, err`?

File: util/util.go [hunk 4]
```diff
@@ -138,6 +142,96 @@ func AbiBigIntTobytes(src *big.Int) []byte {
...
+	}
+
+	for jsonDecoder.More() {
+		arg := consensus.Deploy_Arg{}
+		err := jsonpb.UnmarshalNext(jsonDecoder, &arg)
+		if err != nil {
+			return deployArgs, err
```
Comment (jhyeom26) [inline util/util.go]: same here

Comment (JoowonYun) [reply util/util.go]: https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/9/commits/d7bced484050cbeedfe9b462107a9b460d946c51

File: util/util.go (hunk 3 above)
Comment (JoowonYun) [reply util/util.go]: https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/9/commits/d7bced484050cbeedfe9b462107a9b460d946c51

File: util/util.go (hunk 2 above)
Comment (JoowonYun) [reply util/util.go]: https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/9/commits/d7bced484050cbeedfe9b462107a9b460d946c51

File: util/util.go (hunk 1 above)
Comment (JoowonYun) [reply util/util.go]: https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/9/commits/d7bced484050cbeedfe9b462107a9b460d946c51

File: util/util.go (hunk 2 above)
Comment (jhyeom26) [reply util/util.go]: There is no defined error for this case actually. (The `err` is nil)

Comment (jhyeom26) [reply util/util.go]: How about leaving an issue and proceeding at this time~?

Comment (JoowonYun) [reply util/util.go]: https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/9/commits/d9fee66bb9ed516ecaf49a60f8719310730fd6b4

**Pull request merged**